CONTRIBUTING.md

```diff
@@ -67,7 +67,18 @@ We've included an example env file `test.env.example` with the environment varia
 
 ## Advanced Debugging
 
-If you want to build and test the provider locally there is a make target `make install-tf` that will build the provider binary and install it in a location that terraform can find.
+If you want to build and test the provider locally you should edit you `~.terraformrc` file to include the following:
+
+```
+provider_installation {
+
+  dev_overrides {
+      "registry.terraform.io/Snowflake-Labs/snowflake" = "<path_to_binary>"
```

Comment:
I mean .... you edit the .terraformrc file once and then its good to go. You just run `go install` and it will work. That seems easier than having to remember to run this make command all the time. But I don't mind supporting both methods. I will add back in the commands .